doc/tutorial.md

```diff
@@ -1191,6 +1192,84 @@ worker_defs {
 
 Please refer to doc [tc.md](tc.md).
 
+<a id='acl'/>
+
+# IPset based ACL
+An access control list (ACL) is a list of access control entries (ACE). Each ACE in an ACL identifies a trustee and specifies the access rights allowed or denied for that trustee. DPVS implements an ACL mechanism based on [IPset module](IPset.md). <br>
+Each service contains an ACL rule list where you can add rules. All rules should follow the **PATTERN** of `set=NAME,type={black|white}[,prio={1-100},dir={ingress|egress}]` where `set` and `type` are **essential**. Then you can dynamically change the ACL by adding/deleting entries in the set. For a 'black-type' rule, entries in the set will not be able to establish a DPVS conn. And if a packet is not contained in any 'white-type' rule, it will also be dropped.<br>
```

Comment:
I think more detailed explanations of `type`, `prio`, and `dir` are needed. It should be made clear how does these parameters affect the acl.